CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## 0.160 (Jan 9, 2025)
```

Comment:
Uses Twilio Video Android SDK <Version>

Reply:
oh, the version hasn't been updated to 7.8.0 yet.. will do that once we release Video Android with webrtc-124